.github/workflows/pytest-core-nompi.yml

```diff
@@ -43,55 +44,84 @@ jobs:
           arch: "gcc-5"
           language: "openmp"
           sympy: "1.7"
+          container: false
+          test_suite: "full"
 
         - name: pytest-ubuntu-py38-gcc6-omp
           python-version: '3.8'
           os: ubuntu-18.04
           arch: "gcc-6"
           language: "openmp"
           sympy: "1.8"
+          container: false
+          test_suite: "full"
 
         - name: pytest-ubuntu-py36-gcc7-omp
           python-version: '3.6'
           os: ubuntu-18.04
           arch: "gcc-7"
           language: "openmp"
           sympy: "1.9"
+          container: false
+          test_suite: "full"
 
         - name: pytest-ubuntu-py310-gcc10-noomp
           python-version: '3.10'
           os: ubuntu-20.04
           arch: "gcc-10"
           language: "C"
           sympy: "1.10"
+          container: false
+          test_suite: "full"
 
         - name: pytest-ubuntu-py38-gcc8-omp
           python-version: '3.8'
           os: ubuntu-18.04
           arch: "gcc-8"
           language: "openmp"
           sympy: "1.7"
+          container: false
+          test_suite: "full"
 
         - name: pytest-ubuntu-py39-gcc9-omp
           python-version: '3.9'
           os: ubuntu-20.04
           arch: "gcc-9"
           language: "openmp"
           sympy: "1.8"
+          container: false
+          test_suite: "full"
 
         - name: pytest-osx-py37-clang-omp
           python-version: '3.7'
           os: macos-latest
           arch: "osx"
           language: "C"
           sympy: "1.9"
+          container: false
+          test_suite: "reduced"
 
         - name: pytest-docker-py37-gcc-omp
           python-version: '3.7'
           os: ubuntu-18.04
           arch: "gcc"
           language: "openmp"
           sympy: "1.10"
+          container: true
+          dockerfile: "docker/Dockerfile"
+          image: "devito_img"
+          test_suite: "full"
+
+        - name: pytest-docker-py38-icc-omp
+          python-version: '3.8'
+          os: ubuntu-18.04
+          arch: "icc"
+          language: "openmp"
+          sympy: 1.8
+          container: true
```

Comment:
what's the purpose of the container flag? just set `dockerfile: ""` in the other entries, no?